Apps/APN/App.js

```diff
@@ -0,0 +1,208 @@
+import React, {useEffect, useRef, useState} from 'react';
+import { ActivityIndicator,Linking,StyleSheet} from 'react-native';
+import Login from './components/Login';
+import { NavigationContainer, useNavigationContainerRef } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { CustomerIO, CustomerioConfig, CioLogLevel, CustomerIOEnv } from "customerio-reactnative";
+import Dashboard from './components/Dashboard';
+import CustomDataScreen from './components/CustomDataScreen';
+import SettingsScreen from './components/SettingsScreen'
+import Env from "./env";
+import CioManager from './manager/CioManager';
+import CioKeyValueStorage from './manager/KeyValueStorage';
+import Deeplinks from './components/Deeplink';
+
+const Stack = createNativeStackNavigator();
+
+export default function App() {
+
+  const [firstScreen, setFirstScreen] = useState(undefined)
+  const [loading, setLoading] = useState(true);
+const [isScreenTrackEnabled, setIsScreenTrackEnabled] = useState(null)
+const [isDeviceAttrTrackEnabled, setIsDeviceAttrTrackEnabled] = useState(null)
+const [isDebugModeEnabled, setIsDebugModeEnabled] = useState(null)
+const [bgDelayValue, setBgDelayValue] = useState(null)
+const [bgTasksValue, setBgTasksValue] = useState(null)
+const [trackingUrl, setTrackingUrl] = useState(null)
+
+  useEffect(() => {
+    (async () => {
+      const keyStorageObj = new CioKeyValueStorage()
+      const status = await keyStorageObj.getLoginStatus()
+      setLoading(false)
+      if (JSON.parse(status)) {
+        setFirstScreen("Dashboard")
+        return
+      }
+      setFirstScreen("Login")
+      
+    })();
+  }, [])
+
+  // Automatic screen tracking
+  const navigationRef = useNavigationContainerRef();
+  const routeNameRef = useRef();
+  const config = {
+    screens: {
+      Deeplinks: 'deeplink',
+    },
+  };
+  const linking = {
+    prefixes: ['amiapp://'],
```

Comment:
> I am open to changing it to `apn-sample://`

Since other sample apps are doing this, I think it's a good idea. Although, this is nit picky and not a blocker for this PR.

Reply:
Changed it to `apn-rn-sample` . Will update the same in url scheme as ios native code is in another PR